syntax = "proto3";
package metric_group;




import "google/protobuf/empty.proto";

/*
DeleteMetricGroup请求
*/
message DeleteMetricGroupRequest {
    /*
    套件id
    */
    string pluginId = 1 ;
    /*
    指标组id
    */
    string metricGroupId = 2 ;
}

/*
DeleteMetricGroupApi返回
*/
message DeleteMetricGroupResponseWrapper {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    返回码解释
    */
    string codeExplain = 2 ;
    /*
    错误详情
    */
    string error = 3 ;
    /*
    返回数据
    */
    google.protobuf.Empty data = 4 ;
}

